albumentations/augmentations/functional.py

```diff
@@ -447,21 +447,26 @@ def motion_blur(img, kernel):
 
 
 @preserve_shape
-def jpeg_compression(img, quality):
+def image_compression(img, quality, image_type):
+    quality_flag = cv2.IMWRITE_JPEG_QUALITY
```

Comment:
I would probably make it something like:

```
if image_type == '.jpeg`:
    quality_flag = cv2.IMWRITE_JPEG_QUALITY
elif image_type == '.webp
    quality_flag = cv2.IMWRITE_WEBP_QUALITY
else:
   NotimplementedError("Only '.jpeg' and '.webp' compression transforms are implemented. ")
```

Reply:
It looks better :)